clients/js/test/environmentSetup.ts

```diff
@@ -0,0 +1,40 @@
+import path from "node:path";
+import { GenericContainer, Wait } from "testcontainers";
+import { ChromaClient } from "../src/ChromaClient";
+
+const CHROMADB_PORT = 8000;
+
+export async function startChromaContainer() {
```

Comment:
This is inspired by hosted chroma which uses test containers for postgres and chroma